CONTRIBUTING.md

```diff
@@ -128,7 +131,7 @@ Function declarations are placed in `lib/`, not directly in `server.js`.
 Logos
 -----
 
-Please minimize checked-in SVG files through [SVGO][]. You can use [svgomg][].
+Please minimize checked-in SVG files through [SVGO][]. You can use [svgomg][]. In general we only accept logos that have a corresponding badge on the homepage (e.g. the Eclipse logo because we support service badges for the Eclipse Marketplace), but we may also approve logos for tools widely used by developers (e.g. our Slack logo).
```

Comment:
How about adding something suggesting people not expect this to happen... maybe like, “Don’t count on this. It’s at the discretion of the maintainers.” Or along those lines. My goal is that maintainers feel comfortable saying no.

Reply:
How about now? I've tried to rephrase your suggestion in a slightly more welcoming way, hopefully the meaning is not too diluted!

Reply:
I like the way you phrased it. Welcoming is good. I'm just looking out for maintainers here, and wanting people to feel comfortable saying no.